doc/controls/NavigationBar.md

```diff
@@ -0,0 +1,792 @@
+      </AppBarButton.Icon>
+    </AppBarButton>
+  </utu:NavigationBar.MainCommand>
+</utu:NavigationBar>
+```
+
+
+#### Remarks
+
+Only supports `BitmapImage`
+
+
+## Navigation 
+
+The `NavigationBar` automatically hooks itself up to the  [`SystemNavigationManager.BackRequested`](https://docs.microsoft.com/en-us/uwp/api/windows.ui.core.systemnavigationmanager.backrequested?view=winrt-22000) event and will attempt to navigate back by calling `Frame.GoBack()` as long as all of the following conditions are met:
+* `MainCommandMode` for the `NavigationBar` is set to `MainCommandMode.Back`
+* The `Frame` that contains the `NavigationBar` is currently visible
+* The current `Page` of the `Frame` is equal to the parent `Page` of the `NavigationBar`
+
+NOTE: `SystemNavigationManager` is not supported for WinAppSDK
+
+The `NavigationBar` is also aware of its parent `Page` possibly being hosted in a `Popup` (for things like modal pages) and will close the `Popup` when attempting to navigate backward within the `Popup` while `Frame.BackStack` is empty.
+
+## Placement
+
+On **iOS**, the same `UINavigationBar` instance is shared throughout all pages. When navigating between two pages, you can see that the `UINavigationBar` doesn't move, and only its content and the pages have a transition. To get the same effect for Uno's `NavigationBar`, the `NavigationBar` that you define inside your `Page` is never actually rendered there, and only serves as a placeholder from which to read the information we need to pass to the shared native `UINavigationBar` instance.
+
+To ensure everything works properly, you must follow a few rules:
+
+* The `NavigationBar` must stretch horizontally and be aligned with the top of your `Page`
+* The `NavigationBar` can't move (i.e., don't put it inside a `ScrollViewer`)
+* The `NavigationBar` must be accessible as soon as the `Page` is being navigated to (i.e., don't put it inside a `DataTemplate`)
+* There can only be one `NavigationBar` per `Page`
+
+## Extensibility
+
+The `NavigationBar` is automatically managed by the `Frame` control, however you can still use the "native" mode of the `NavigationBar` with your own navigation mechanism.
+
+On **iOS** a `NavigationBarHelper` is available for this purpose, you only have to invoke each of the provided methods in your own `UIViewController` implementation.
+
+## AppBarButton
+
+The `AppBarButton` in **Uno** is designed to be used the same way you would use the `AppBarButton` on **UWP**. In most cases, you should refer to the [official `AppBarButton` documentation](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.appbarbutton).
+
+When `AppBarButton` is used within a native `NavigationBar`, its control template is completely ignored and can't be customized.
+
+### Events
+
+| Event   | Windows | iOS | Android | Comments |
+|---------|:-------:|:---:|:-------:|----------|
+| Clicked | x       | x   | x       |          |
+
+### Properties
```

Comment:
the properties headers should be a tier lower and this header; they are currently on same tier.